docs/game_sdk/Store.md

```diff
@@ -0,0 +1,454 @@
+      "price": {},
+      "premium": false,
+      "locales": ["en-US"],
+      "bundled_skus": null
+    }
+  ]
+}
+```
+
+## Consume SKU % POST /store/applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements/{entitlement.id#DOCS_GAME_SDK_STORE/data-models-entitlement-struct}/consume
+
+Marks a given entitlement for the user as consumed, meaning it will no longer be returned in an entitlements check. **Ensure the user was granted whatever items the entitlement was for before consuming it!**
+
+###### Example
+
+```
+curl -X POST https://discordapp.com/api/v6/applications/461618159171141643/entitlements/53908232506183999/consume \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  "code": 0,
+  "message": "200: OK"
+}
+```
+
+## Get User Entitlements % GET /users/@me/applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements
+
+> warn
+> This endpoint falls under the `/users/@me` route schema, which means that it takes the OAuth2 bearer token of a player. This can be retrieved via the Application Manager in the SDK
+
+Returns the entitlements a user has for your application.
+
+###### Example
+
+```
+curl https://discordapp.com/api/v6/users/@me/applications/461618159171141643/entitlements \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  [
+    {
+      "user_id": "53908232506183680",
+      "sku_id": "53908232599983680",
+      "application_id": "461618159171141643",
+      "id": "53908232506183999",
+      "type": 3
+    }
+  ]
+}
+```
+
+## Checking DLC Entitlements
+
+If your game has DLC, and a user has purchased that DLC, you may want to check what they should have access to when the game launches. DLC entitlements will always be returned in a `FetchEntitlements()` call, so your game can check on each startup whether or not a user should have access to a certain new zone, raid, map, etc. based on their entitlements for DLC.
+
+Entitlements to DLCs are intended to be for SKUs that will download additional data onto a user's hard drive, like an expansion pack for a game.
+
+## Checking Consumable Entitlements
+
+The `Discord.SkuType.Consumable` type is used for entitlements that may be "consumed" by a game's own server infrastructure. That is to say that if you have in-app purchases like gem bundles, skins, etc., they will be a `Consumable` SKU type.
+
+What that means is that your game is expected to consume these entitlements and then tell Discord that they've been consumed. Your game should handle these granted consumable entitlements in whatever manner is appropriate: incrementing the coins a player has, flipping the flag for a certain skin, etc.
+
+Entitlements to consumable SKUs are intended to be for SKUs that do not download additional content to a user's machine; rather, they signal to your game's server/service/database that the user should get something new and should be incremented as such.
```

Comment:
It may be worth noting that consumables can be purchased multiple times others cannot.